### ISSUE TITLE
Feature request: Support Japanese Characters

### ISSUE THREAD

**Issue by gajewsk2** [chanind/hanzi-writer-data#2]
Would it be possible to load the data mentioned here: https://github.com/skishore/makemeahanzi/issues/18
into the hanzi writer data for use with the hanzi writer library.

**Comment by chanind** [comment]
ack I'm really sorry I somehow missed this comment! Those do look compatible with Hanzi Writer. Are they not already in this data repo, or is it different variations on the same characters that already exist?

**Comment by chanind** [comment]
Apologies for the extreme delay on this. I put a repo up with Japanese data from [animCJK](https://github.com/parsimonhi/animCJK) here: https://github.com/chanind/hanzi-writer-data-jp. You can use this in Hanzi Writer via a custom `charDataLoader` function, like below:

```
HanziWriter.create('target-div', '私', {
  width: 400,
  height: 400,
  charDataLoader: (char, onLoad, onError) => {
    fetch(`https://cdn.jsdelivr.net/npm/hanzi-writer-data-jp@0/${char}.json`)
      .then(res => res.json())
      .then(onLoad)
      .catch(onError);
  }
})
```

This Japanese data is limited compared to the Chinese currently due to:
- The data does not support radicals yet
- The data does not have capped strokes, so there are sharp edges where strokes intersect

I'll work on tidying this up before making an official release in HanziWriter adding first-class support for this data. I also want to double-check with animCJK that using the data this way is alright.

**Comment by Anrai** [comment]
Hello, author. Thank you very much for your works, It is great! but I have some problems now. Some Chinese characters do not exist in your hanzi-writer-data@2.0, May I ask if it is possible to increase? Thank you!

**Comment by JhonnySalles** [comment]
@Anrai 

To get around this problem, I do the query on the Japanese IP first, if there is an error to load, the function **HanziWriter.create** executes a new call on the **onLoadCharDataError** but on the Chinese IP.

With that I got around the problem of not finding the character, when it doesn't find a small leg at first, but it works normally.

I believe that the correct thing would be to increase the number of characters in Japanese, but as long as it does not happen you can solve the problem.

The following example images.

![image](https://user-images.githubusercontent.com/43912638/85165231-e3a0c400-b23b-11ea-9bcb-fb7a6bfcec5e.png)

![image](https://user-images.githubusercontent.com/43912638/85165754-ab4db580-b23c-11ea-86ff-220829cbdf3e.png)

**Comment by chanind** [comment]
This is a great idea, thanks for sharing! Hopefully we can get something like this built-in to Hanzi Writer so it's easier to work with Japanese characters without needing to do these workarounds.

**Comment by JhonnySalles** [comment]
Yes, it was a good change, there is only the problem of delay between loading when there is no kanji in the Japanese set. Despite this 1 or 2 second delay, everything works correctly.

From what I realized there are very few that do not exist, but studying the JLPT from 1 to 5, I noticed that there are some missing, even containing 2 thousand kanji.

I have a dictionary that works with javascript that presents popup for kanji. The name is "_kanjax_with_koohii", it has 3,000 kanji in it, you could take them into account to import Chinese kanji to Japanese.

![image](https://user-images.githubusercontent.com/43912638/85321643-62476c80-b49b-11ea-8bdc-6b09954ba024.png)

![image](https://user-images.githubusercontent.com/43912638/85321656-670c2080-b49b-11ea-9a3a-17fd38a91cb1.png)
